src/nccl_ofi_rdma.c

```diff
@@ -4830,14 +4830,14 @@ static int send_close(nccl_net_ofi_send_comm_t *send_comm)
 		goto exit;
 	}
 
-	nccl_net_ofi_rdma_ep_t *ep = (nccl_net_ofi_rdma_ep_t *) s_comm->base.base.ep;
```

Comment:
This change has nothing to do with the string concatenation change.

Reply:
missed this when rebasing, thanks